api/hasura/cron/_epochs.test.ts

```diff
@@ -1,5 +1,5 @@
 import faker from 'faker';
```

Comment:
can you merge this file with the one in `api-test/hasura`?

Reply:
Actually @levity we can't/shouldn't merge these files because the gql client is stubbed in these tests but I can colocate it with a more descriptive rename